### PR TITLE
Add notifications card

### PR DESCRIPTION
**Pull request by nimesh-irisio**

### CODE REVIEW

**Comment by nimesh-irisio** [comment]
Closing this one out to move this over to Account View per @SebastianBor feedback!

File: VoteStakeRegistry/components/Account/LockTokensAccount.tsx [hunk 1]
```diff
@@ -362,6 +364,11 @@ const LockTokensAccount = ({ tokenOwnerRecordPk }) => {
         )}
       </div>
       {connected && <Account withHeader={false}></Account>}
+      {realmInfo?.enableNotifications && (
```
Comment (nimesh-irisio) [inline VoteStakeRegistry/components/Account/LockTokensAccount.tsx:367]: @SebastianBor  Assuming this is how you wanted to enable/disable notifications feature for a realm.

Comment (SebastianBor) [reply VoteStakeRegistry/components/Account/LockTokensAccount.tsx]: Yes, thank you, however the feature toggle name should be more specific to your product: `enableNotifi`
It's the only notification option atm so I think it's ok for now to have it enabled by default for better discoverability and to give DAOs a chance to try it.
Please change the name and enable it for all realms and it's good to go

Comment (nimesh-irisio) [reply VoteStakeRegistry/components/Account/LockTokensAccount.tsx]: Pushed an update to address this comment and add the footer for "learn more"

**Pull request merged**